### PR TITLE
test case: manual: pulseaudio test case

### PR DESCRIPTION
**Pull request by ClarexZhou**
Signed-off-by: clarex <clarex.zhou@intel.com>

Add more test cases for pulseaudio

### CODE REVIEW

File: test-case/manual/check-pulseaudio-automatic-source-switch.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+# Test Description
+* Output source and intput source should be automatic changed once connect or disconnect headset
+* Output source should be automatic changed once connect or disconnect HDMI
+
+## Headset Test Case:
+1. Boot up system without HDMI and headset connection
+2. `parecord 1.wav -vvvv`, build-in device, such as DMIC is used for capturing. Say something then ctrl+c to stop
+3. `paplay 1.wav -vvvv`, internal speaker is used for playback. Audio is output successfully then press ctrl+c to stop
+4. Wait for about 3s till runtime PM is suspended, can check via `cat "/sys/bus/pci/devices/0000:00:1f.3/power/runtime_status"`
```
Comment (Bin-QA) [inline test-case/manual/check-pulseaudio-automatic-source-switch.md:16]: you need to point this pci id `00:1f.3` is for intel audio device from `lspci`

Comment (ClarexZhou) [reply test-case/manual/check-pulseaudio-automatic-source-switch.md]: updated: 4. Wait for about 3s till runtime PM is suspended, can check via `cat "/sys/bus/pci/devices/0000:00:1f.3/power/runtime_status"`, the pci device id can be find from `lspci` for Multimedia audio controller

File: test-case/manual/check-pulseaudio-automatic-source-switch.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+2. Plug in HDMI/DP
+3. Open sound setting, check Output source is auto switched to HDMI/DP. Close sound setting
+4. Plug in headset
+5. Open sound setting, Output source is auto switched to headset, Input source is headset MIC
+6. Unplug headset, both output source and input source are auto switched
+7. Plug in headset, both output source and input soruce are auto switched
+8. Unplug HDMI, output source and input source isn't changed.
+9. Plug in HDMI again, output soruce is auto switched to HDMI/DP, Input source is DMIC
+10. `parecord 3.wav -vvvv`, DMIC is used for capturing. Say something then ctrl+c to stop
+11. `paplay 4.wav -vvvv`, internal speaker is used for playback normally
```
Comment (fredoh9) [inline test-case/manual/check-pulseaudio-automatic-source-switch.md:59]: I can easily follow up to 10. For 11, It should be

`11. `paplay 3.wav -vvvv`, HDMI is used for playback normally`

Comment (ClarexZhou) [reply test-case/manual/check-pulseaudio-automatic-source-switch.md]: thanks and updated

File: test-case/manual/check-pulseaudio-automatic-source-switch.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+2. Plug in HDMI/DP
+3. Open sound setting, check Output source is auto switched to HDMI/DP. Close sound setting
+4. Plug in headset
+5. Open sound setting, Output source is auto switched to headset, Input source is headset MIC
+6. Unplug headset, both output source and input source are auto switched
+7. Plug in headset, both output source and input soruce are auto switched
+8. Unplug HDMI, output source and input source isn't changed.
+9. Plug in HDMI again, output soruce is auto switched to HDMI/DP, Input source is DMIC
+10. `parecord 3.wav -vvvv`, DMIC is used for capturing. Say something then ctrl+c to stop
+11. `paplay 4.wav -vvvv`, HDMI is used for playback normally
```
Comment (fredoh9) [inline test-case/manual/check-pulseaudio-automatic-source-switch.md:59]: everything else is good.
```suggestion
11. `paplay 3.wav -vvvv`, HDMI is used for playback normally
```

**Pull request merged**